docs/src/main/sphinx/connector/hive.md

```diff
@@ -315,7 +315,7 @@ the catalog properties file. This property must be one of the following values:
 (hive-sql-standard-based-authorization)=
 ### SQL standard based authorization
 
-When `sql-standard` security is enabled, Trino enforces the same SQLuyrity
+When `sql-standard` security is enabled, Trino enforces the same SQL 
```

Comment:
Please remove a following space. 

Also, no need to add the commit body as pretty simple change.